sparta/sparta/utils/BitArray.hpp

```diff
@@ -220,6 +220,7 @@ namespace sparta {
 
                 if (data_size > SMALL_OPTIMIZATION_SIZE) {
                     data_ls_.reserve(data_size);
+                    std::memset(&data_ls_[0], 0, data_size * sizeof(data_type));
```

Comment:
Instead of reserving the memory, resize the vector with the value 0.

Reply:
How could I forget this function? Thanks!